packages/fxa-content-server/app/scripts/lib/app-start.js

```diff
@@ -381,15 +350,6 @@ Start.prototype = {
     }
   },
 
-  initializeFxaClient () {
```

Comment:
I believe both the OAuthClient and the FxaClient need to still be created here because they are used by the views and some of the models.

Reply:
Okay, I'll add them back